### ISSUE TITLE
Update accessible balloons and static electricity links

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/website#514]
From https://github.com/phetsims/balloons-and-static-electricity/issues/174, for the Assets conference, it would be nice if the links to accessible balloons and static electricity are updated to the latest version.  The following links would need to be updated:
1.2.0-accessible-instance.16
![base1](https://cloud.githubusercontent.com/assets/6396244/19493024/331b036e-9536-11e6-9312-16737af67bb6.PNG)

1.2.0-accessible-instance.14
![base2](https://cloud.githubusercontent.com/assets/6396244/19493026/34d5cfb8-9536-11e6-9705-f4980f010cb2.PNG)

@mattpen would it be possible to have the links updated by Monday 10/24/16?  I will have the latest dev version up by 10/20/16.

**Comment by mattpen** [comment]
@jessegreenberg - Yes, this shouldn't be a problem.  Please post where these links should now point and what the new text should be.  Please let me know before 10/21.

**Comment by jessegreenberg** [comment]
Thanks @mattpen, I will have the link posted in this issue by the end of today 10/20.

**Comment by jessegreenberg** [comment]
@mattpen here is the destination which both links should now point to:
http://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.2.0-reader.7/balloons-and-static-electricity_en.html

Could you also please rename both links to ```1.2.0-reader.7```?  Thanks!  Please let me know if you need any more information.

**Comment by mattpen** [comment]
@jessegreenberg - These should be up on the main website now.  Please confirm and close if it looks ok.

**Comment by jessegreenberg** [comment]
Thanks @mattpen, the links are working great.  It sounds like there are a couple more things that we would like to add to the accessibility page:
- A link to an accessible version of the poster that @terracoda is using to present at the assets conference.  
- A link to an accessible document describing screen reader compatibility of Balloons and Static Electricity.

We will post links to the content in this issue before this evening with details of where they should be located on the page.

**Comment by terracoda** [comment]
@jessegreenberg and @mattpen, FYI, the poster PDF is not an accessible PDF. The word document version is the only accessible content at the moment.

**Comment by jessegreenberg** [comment]
@mattpen if you have time, here is the list of requested changes:
- [ ] Under 'Auditory Descriptions', under the compatibility table for Balloons and Static Electricity, can you please add the following note:  ```'Note:  This prototype is also compatible with VoiceOver using the Chrome browser."```
- [ ] Add a link to the PDF of the poster, and then a link to the accessible Word version to the listing for the ASSETS conference in "Upcoming Meetings".  For example (with the links):
  - October 24-26, 2016 - Taliesin L. Smith, Emily B. Moore, and Clayton Lewis. Demonstration: Screen reader support for a complex interactive science simulation. ACM SIGACCESS ASSETS Conference. Reno, NV. Poster (PDF) (Word) 
[PDF link: https://drive.google.com/file/d/0B44Uycdx6JGdRFpXcDJqZl9BUk0/view?usp=sharing]
[Word Link: https://drive.google.com/file/d/0B44Uycdx6JGdVFJNUU9QQUpRRnM/view?usp=sharing]

- [ ]  *If you have time -* Move the following "Upcoming Presentations" to "Past Presentations" (this content should be moved):
  - July 19-22, 2016 - Taliesin L. Smith, Clayton Lewis, and Emily B. Moore. A balloon, a sweater, and a wall: Developing design strategies for accessible user experiences with a science simulation. International Conference on Universal Access in Human-Computer Interaction. Toronto, Canada. 

  - July 19-22, 2016 - Emily B. Moore, Taliesin L. Smith, and Emily Randall. Exploring the relationships between implicit scaffolding and inclusive design in interactive science simulations. International Conference on Universal Access in Human-Computer Interaction. Toronto, Canada. 

  - July 20-21, 2016 - Elise Morgan and Emily B. Moore. Investigating student learning with accessible interactive physics simulations. Physics Education Research Conference. Sacramento, CA. Poster 
[Poster Link = https://drive.google.com/file/d/0B44Uycdx6JGdUk1oTU9GNC13VkU/view?usp=sharing].

  - July 16-20, 2016 - Elise Morgan and Emily B. Moore. Exploring best practices in accessible design of interactive science simulations. AAPT Summer Meeting. Sacramento, CA. 

  - June 1-3, 2016 - Emily B. Moore, Katherine Perkins, Jutta Treviranus. Ramping up accessibility in STEM: Inclusively designed simulations for diverse learners. NSF DRK-12 PI Meeting. Washington, DC. Poster
[Poster Link = https://drive.google.com/file/d/0B44Uycdx6JGdT3I0UmZBUGR4TGc/view?usp=sharing]

  - May 17-23, 2016 - Emily B. Moore, Accessible PhET simulations for diverse learners. NSF STEM For All Video Showcase. Virtual Showcase. Video.

Please let me know if you have any questions.  Also pinging @emily-phet in case she would like to review this list.

**Comment by mattpen** [comment]
@jessegreenberg - These were deployed to the website tonight.  Please let me know if any further changes are needed.

**Comment by terracoda** [comment]
Hi @mattpen, I saw a content issue (not critical) in the list of keys for BASE. The H key only bring up help when the user has the balloon grabbed. We could change:

Press 'H' anywhere to pull up a keyboard help dialog.
to
Once balloon is grabbed, the 'H' key is pressed to pull up a keyboard help dialog.

**Comment by emily-phet** [comment]
@mattpen - Here's a tweak to the wording @terracoda suggests above:

Instead of "Press 'H' anywhere to pull up a keyboard help dialog." replace with:
"Once balloon is grabbed, press 'H' key to access the keyboard help dialog."

To make this consistent across the other instructions also replace "Once a balloon is picked up, it can be dragged with the 'WASD' keys" with:
"Once a balloon is picked up, press the 'W', 'A', 'S', or 'D' keys to drag."

**Comment by jessegreenberg** [comment]
@mattpen sorry I didn't see this issue.  Would you mind making the changes above whenever it is convenient to deploy the website?

**Comment by mattpen** [comment]
If this is not time sensitive, perhaps it can wait until the admin interface is working? Referring to https://github.com/phetsims/website/issues/533

**Comment by jessegreenberg** [comment]
@emily-phet @terracoda would https://github.com/phetsims/website/issues/514#issuecomment-263339606 be OK?

**Comment by emily-phet** [comment]
@mattpen - the admin interface discussed in #533 is referring to changes in the "Research and Design" page. The suggestions above are changes to the "Prototypes" page. Could you go ahead and make the changes described above to the prototypes page?
